### PR TITLE
AnP Create: Now with More Interactivity!

### PR DESCRIPTION
**Pull request by tmooney**
Here's an alternative proposal to address #2000 that adds a prompt and a bit more text about interacting with the chooser.

### CODE REVIEW

File: lib/perl/Genome/Config/AnalysisProject/Command/Create.pm [hunk 1]
```diff
@@ -125,12 +125,25 @@ sub _get_menu_items {
     } elsif ($self->no_config()) {
         return [];
     } else {
+
+        my $add_config = $self->_ask_user_question(
+            'Would you like to add configuration from the preset config menu items?',
+            60,
```
Comment (chrisamiller) [inline lib/perl/Genome/Config/AnalysisProject/Command/Create.pm:131]: what does the 60 here do?

**Review by chrisamiller — APPROVED** [review]
I like this a lot and think it makes things more user friendly

File: lib/perl/Genome/Config/AnalysisProject/Command/Create.pm (hunk 1 above)
Comment (tmooney) [reply lib/perl/Genome/Config/AnalysisProject/Command/Create.pm:131]: It's the "timeout".  If you don't answer in 60 seconds, it `exit`s.  (That's actually the default in `_ask_user_question`, but made explicit here.)  Can also use `0` for unlimited time.

Comment (chrisamiller) [reply lib/perl/Genome/Config/AnalysisProject/Command/Create.pm:131]: got it! 👍

**Pull request merged**